legacy-app/src/app/modules/events/pages/event-details-page/event-details-page.component.ts

```diff
@@ -45,13 +47,16 @@ export class EventDetailsPageComponent implements OnDestroy {
   private loadEventQueryRef;
   private destroyed$ = new Subject();
 
+  public ratingExpanded = false;
```

Comment:
I think a BehaviourSubject would be the better choice here in case we us onPush change detection